include/tscore/Ptr.h

```diff
@@ -99,10 +99,12 @@ template <class T> class Ptr
 public:
   explicit Ptr(T *p = nullptr);
   Ptr(const Ptr<T> &);
+  Ptr(Ptr<T> &&);
```

Comment:
The changes to this file are done to address a mistaken clang-analyzer issue that gets raised from this patch. (Thank you, @ywkaras, for your work on coming up with this solution.)